internal/pkg/form/form.go

```diff
@@ -112,6 +112,20 @@ func ConfirmDeletion(cmd *cobra.Command, promptMsg, stringToType string) (bool,
 	return false, errors.NewErrorWithSuggestions(fmt.Sprintf(`input does not match "%s"`, stringToType), DeleteResourceConfirmSuggestions)
 }
 
+func ConfirmEnter(cmd *cobra.Command) (error) {
+	// Note: this function does not check against any regex;
+	// any string is acceptable
+	prompt := NewPrompt(os.Stdin)
+	f := Field{Prompt: "Press enter to continue or Ctrl-C to cancel"}
```

Comment:
Can we hide the output from the user? If they type "no" for example they shouldn't see any letters appear.

Reply:
I explored this for a bit. Using `golang.org/x/term`'s ReadPassword function to disable terminal echo does this. But... it has an unexpected interaction with `browser.OpenURL` that causes the tab to open in the background. Going to make a ticket for this and address it afterwards, but I want to get this in for the next release.

Reply:
Huh, that's super weird.

Reply:
Added it back. Printing anything gets rid of this behavior.